CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+1.4.0 / 2019-08-28
+==================
+
+###### Changes
+* Dropping support for PHP 5.3
+* Updated documentation about testing
+* Syn library is [0.0.2](https://github.com/bitovi/syn/tree/v0.0.2)
+* `setDesiredCapabilities` will throw an exception if you will try to set capabilities on already started session
+* Added `getDesiredCapabilities` to allow fetch capabilities
+* Minors improvements to driver when running against W3C compatible driver
+ 
+###### Testsuite
+* Adding testing on PHP 7.1, 7.2, 7.3 and 7.4beta
```

Comment:
Please also add these changes:

```
* Removed PhantomJS support
```

Reply:
it's not correct. PhantomJS may still work and is working (In my company we still are running PhantomJS). We've just removed it from our testing suite.